install.sh

```diff
@@ -98,18 +98,30 @@ git clone -o cms-hh git@github.com:cms-hh/Resources.git HHStatAnalysis/Resources
 GITHUB_USER=$(git config user.github)
 if ! [ "x$GITHUB_USER" = "x" ] ; then
     cd HHStatAnalysis
-    git remote add $GITHUB_USER git@github.com:$GITHUB_USER/HHStatAnalysis.git
-    git fetch $GITHUB_USER
+    git ls-remote git@github.com:$GITHUB_USER/HHStatAnalysis.git &> /dev/null
```

Comment:
I'm not a fan of putting stiff to `/dev/null` until necessary.. what is the reason here ? there seems to be a `--quiet` option for `git ls-remote` ....